public/static/locales/en/header.json

```diff
@@ -10,5 +10,8 @@
   "listings": "Listings",
   "home": "Home",
   "travel": "Travel Directory",
-  "mobile": "Back to homepage"
+  "mobile": "Back to homepage",
+  "bannerHeading": "We're moving to MoneyHelper",
+  "banner" :"The PensionWise service will soon be part of MoneyHelper, the easy way to get free help for your money and penions.",
```

Comment:
Can we check the copy here? This appears to be the copy for Pension Wise where (I guess) we need the copy for MAS. (Also 'pensions' is misspelled)

Reply:
Fixed.